ui/.github_release_version

```diff
@@ -1,3 +1,3 @@
 # This file contains a version number which will be used to release assets to
 # GitHub. To trigger a new asset release, simply increase this version number.
-20200214_1
+20200218_1
```

Comment:
Will we need a special process to modify this line in the future?

Reply:
Nop, it's a specially designed as a manual process so that release is only run on demand.